dev/bin/configure.sh

```diff
@@ -37,17 +37,36 @@ for tool in $required_tools; do
   which "$tool" &> /dev/null || die "required tool \"$tool\" is missing"
 done
 
+# export useful paths
+export ROOT_DIR=`realpath ${CONFIGURE_SCRIPT_DIR_PATH}/../..`
+
+export DEV_DIR="${ROOT_DIR}/dev"
+export LOCAL_DEPLOY_DIR="${DEV_DIR}/_deploy"
+export DEV_CONFIG_DIR="${DEV_DIR}/config"
+export DEV_BIN_DIR="${DEV_DIR}/bin"
+export DEV_LOGS_DIR="${DEV_DIR}/logs"
+export DEV_DOCKER_COMPOSE_DIR="${DEV_DIR}/docker-compose"
+export CREDENTIALS_DIR="${DEV_DIR}/credentials"
+export KUSTOMIZE_DEPLOY_DIR="${LOCAL_DEPLOY_DIR}/kustomize"
+
+export LOCAL_ENV_FILE="${LOCAL_DEPLOY_DIR}/local_env"
+export LOCAL_CONFIG_FILE="${DEV_CONFIG_DIR}/localconfig"
+
+export KUSTOMIZE_DIR="${ROOT_DIR}/kustomize"
```

Comment:
Why are we exporting all these variables?

Reply:
Also, we should use lowercase names for local variables and uppercase only for environment variables to avoid conflicts (see https://stackoverflow.com/a/673940)

Reply:
those are meant to be used in other scripts. so they should be considered as env vars. that is why we are exporting them

Reply:
But all other scripts are sourcing `configure.sh`, so these variables will be already set without exporting them (which increases the risk of conflicts with existing env variables).

I would use uppercase variables only for actual env variables that the user can use to configure the scripts (e.g. `MANAGED_KAFKA_INSTANCE_NAME`).

Reply:
I updated all of those. we now use only lowercase names for variables coming from `configure.sh` (renamed to `common.sh`)